scripts/packageTorClient.js

```diff
@@ -20,7 +20,7 @@ const downloadTorClient = (platform) => {
   const torS3Prefix = process.env.S3_DEMO_TOR_PREFIX
 
   const torVersion = '0.4.5.10'
```

Comment:
No need to update this version number?

Reply:
We are just updating the OpenSSL library, tor version remains the same. Updated the `braveVersion` here: https://github.com/brave/brave-core-crx-packager/pull/254/files#diff-50089866665384155d804130c89c80a81693797ea034d0786391b720c16a9c7dR23